pal/screen-adapter/minigame/screen-adapter.ts

```diff
@@ -51,6 +51,7 @@ try {
         }
     }
 } catch (e) {
+    // eslint-disable-next-line no-console
```

Comment:
Don't do it like this. It just avoid error. I fixed in develop branch, so don't have to do in v3.8.1.